test/Jsr292/src/com/ibm/j9/jsr292/indyn/IndyTest.java

```diff
@@ -33,6 +33,13 @@
 
 public class IndyTest {
 
+	private static final String JAVA_VERSION = System.getProperty("java.version");
+	
+	/* Return true if Java 8 is used; otherwise, return false. */
+	private boolean isJava8() {
+		return JAVA_VERSION.startsWith("1.8.0");
+	}
+
```

Comment:
It is likely that many tests will use similar isJava8(), isJava9() utility functions, seems like something we may want to add to test/TestUtilities dir